proj_utils/Parser.py

```diff
@@ -0,0 +1,75 @@
+from time import sleep
+
+from models.Payload import Payload
+import os
+import json
+
+
+class Parser:
+    def __init__(self, payload: Payload):
+        self.payload = payload
+        self.jsonObject = {}
+
+    def create_text_file(self):
+        save_path = 'C:/Users/razm1/PycharmProjects/routeExercise/students'
```

Comment:
this line has two problems.
1. this line is an absolute path, which means it can only run if the whole path exists, which is probably only on your computer.
2. it is hard coded. it should be configurable